index.html

```diff
@@ -373,17 +376,22 @@ <h3>Use Cases and Requirements</h3>
 to help them manage and share their <a>verifiable credentials</a>.
         </li>
         <li>
-<a>Holders</a> that share <a>verifiable credentials</a> are not required to
+<a>Holders</a> that share <a>verifiable presentations</a> are not required to
 reveal the identity of the <a>verifier</a> to <a>issuers</a>.
         </li>
         <li>
-A <a>verifiable credential</a> is expressed in one or more standard,
-machine-readable data formats which can also be extended with minimal
-coordination.
+<a>Holders</a> in <a>presentations</a> can either disclose the attributes or satisfy <a>predicates</a>
```

Comment:
predicates are linked to here, but not defined. If we are going to introduce that new term, we'll have to define it in terms.html

Reply:
Thinking about it a bit more, do we need to introduce the "predicate" term? It's just another attribute isn't it? Can't we just provide an example of an attribute that is a predicate w/o increasing the cognitive burden of the reader to have to learn a new term?

Reply:
The way I've been using it, a "predicate" (I've now renamed to "derived predicate" in latest incarnation of PR) is NOT an attribute in a credential. It is an attribute in a presentation that is derived from an attribute in a credential.